lib/dealership.rb

```diff
@@ -0,0 +1,47 @@
+class Dealership
+  attr_reader :name, :address, :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @inventory_count = 0
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def inventory_count
+    @inventory_count = @inventory.length
+  end
+
+  def has_inventory?
+    if @inventory_count < 2
```

Comment:
How many cars would our `inventory_count` need to have in it to return `false`? If we had 0 cars would we have any inventory? So what would we want `inventory_count` to be greater than?